### ISSUE TITLE
Serve an "info" page for a Built Binder

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/binderhub#565]
There have been a few moments where people want access to meta information about a Binder from within a live session. A few that come to mind:

1. What is the "sharable link" for a Binder repository?
2. What is the github repo link that a Binder repository was built from?
3. Where are the logs for the build process?
4. Something @bitnik implemented - what is the token for my private binder repo so that I can _share_ it with others

What if we created an `info/` page that was generated for each Binder repository. This could live at `https://hub.mybinder.org/user/<pod-name>/info` and would be a simple page with auto-generated HTML. Perhaps this could be a way to provide this information for people, without making a custom UI edit to the notebook/lab/rstudio/etc UI. What do folks think?

**Comment by betatim** [comment]
Making the build log available would be good and a new page would be the way to go.

The sharing links should be in the UI IMHO. So either as notebook extension or iframe around the content.

I don't think we should encourage people to share their private binder links, so I'd not have a button to copy&paste that URL. People would probably use it accidentally. There is already far too much confusion about which link you want to share when.

**Comment by willingc** [comment]
I agree with all except I do not think, at this time, we should officially support private tokens or repos. Better guidance would be to direct those using private repos to create their own binderhub.

**Comment by choldgraf** [comment]
That sounds good to me!

**Comment by betatim** [comment]
Chris could you clarify what you meant with "what is the token for my private binder repo so that I can share it with others". I took it as "how to share a live binder instance (made from a public repo)". Reading Carols comment I am now not so sure any more. If there is a way to use private git repositories with mybinder.org right now I'd consider that a bug we need to fix.

**Comment by choldgraf** [comment]
In this case I meant "sharing the notebook token for your session so that other people can connect to it". Not "using private github repos". I think @willingc and you make a good point re: avoiding sharing personal session tokens, so I'm happy not to have that in there.

**Comment by willingc** [comment]
@choldgraf I'm curious :-) What is the use case that prompted wanting to share a session token? Was the user's notebook accessing data that was not publicly available? Collaboration? ???

**Comment by choldgraf** [comment]
@willingc yep, I believe so. I think the GESIM folks are already using this. Basically, they have a button at the top of their page that, when clicked, gives you the URL *to your live session* that you can share with others. Presumably for collaboration and sharing live sessions. I think that deployment has different security concerns (e.g. it's authenticated, I believe), but I thought it was a very interesting idea
